src/creature_states_spdig.c

```diff
@@ -1553,6 +1609,56 @@ short creature_pick_up_unconscious_body(struct Thing *thing)
     return 1;
 }
 
+short creature_save_unconscious_creature(struct Thing *thing)
+{
+    SYNCDBG(9,"Starting");
+    TRACE_THING(thing);
+    // Check if the player has means to do such kind of action
+     if (!player_has_room_of_role(thing->owner, RoRoF_LairStorage) || !player_creature_tends_to(thing->owner, CrTend_Flee))
```

Comment:
why wouldn't it apply when they flee? if the battle is done, don't see how flee is relevant here

Reply:
I do not think people ever want their creatures _not_ saved when it an option to save them right?

Reply:
hm... so if bugs or flies lie inside the enemy territory and your imps die because they want to save them?

Reply:
You'd normally want to make vampires of your enemies, not your friends. And yeah, you dislike imps going to dangerous territory, but the same happens now with graveyards or claiming or prisons. And yes, you could handle those the way you say, but that's not usually what people would do anyway. I very rarely if ever see people lock away a graveyard, and locking out the enemies themselves with a door, or your imps in a locked room seem more obvious solutions there.